github-merge.py

```diff
@@ -125,6 +125,8 @@ def tree_sha512sum(commit='HEAD'):
     for line in subprocess.check_output([GIT, 'ls-tree', '--full-tree', '-r', commit]).splitlines():
         name_sep = line.index(b'\t')
         metadata = line[:name_sep].split() # perms, 'blob', blobid
+        if metadata[1] == b'commit':
+            break # Ignore submodule
```

Comment:
Can you explain why break is used instead of continue?

Reply:
Switched to continue, that makes more sense. But I'm not even sure if skipping is the right move, or whether we need to just process a `b'commit'` differently.